flow360/component/simulation/simulation_params.py

```diff
@@ -229,19 +229,6 @@ def apply_defult_fluid_settings(cls, v):
             v.append(Fluid())
         return v
 
-    @pd.field_validator("outputs", mode="after")
-    @classmethod
-    def apply_defult_output_settings(cls, v):
-        """[Solver Capability Related] apply default SurfaceOutput settings if not found in outputs"""
-        if v is None:
-            v = []
-        assert isinstance(v, list)
-        if not any(isinstance(item, SurfaceOutput) for item in v):
-            v.append(
-                SurfaceOutput(name="Surface output 1", output_fields=["Cp", "yPlus", "Cf", "CfVec"])
-            )
-        return v
-
```

Comment:
can you write in the description of PR why this cannot be done this way anymore?